dart/lib/src/sentry_client.dart

```diff
@@ -99,6 +100,15 @@ class SentryClient {
       event = event.copyWith(exception: sentryException);
     }
 
+    if (_options.attachStackTrace &&
+        event.throwable == null &&
+        event.exception == null) {
+      final frames =
+          SentryStackTraceFactory(_options).getStackFrames(StackTrace.current);
```

Comment:
please find a way to reuse the instance of `SentryStackTraceFactory`.

`SentryExceptionFactory` already creates an instance of it, it'd be easier to inject/test and also to not allocate memory on each call to captureEvent

Reply:
✅